src/net_gen/src/if_tun.rs

```diff
@@ -9,7 +9,8 @@
     dead_code,
     non_snake_case,
     clippy::ptr_as_ptr,
-    clippy::undocumented_unsafe_blocks
+    clippy::undocumented_unsafe_blocks,
+    missing_debug_implementations
```

Comment:
In `.cargo/config` it sets it to deny, in the other files which are bindings it sets it to allow (same as with the other lints).

If we remove these allows for the auto-generated bindings it produces a bunch of errors (see below).

The only way to avoid duplication would be to generate the bindings at compile time to avoid including the files under `/src` like https://rust-lang.github.io/rust-bindgen/library-usage.html. I would prefer this approach, but this is another topic (and one I believe I previously lost this argument 😅)
```
root@a71d948ddcbf:/firecracker# cargo clippy --all --tests
warning: patch for `kvm-bindings` uses the features mechanism. default-features and features will not take effect because the patch dependency does not support this mechanism
    Checking net_gen v0.1.0 (/firecracker/src/net_gen)
   Compiling seccompiler v1.5.0-dev (/firecracker/src/seccompiler)
   Compiling vmm v0.1.0 (/firecracker/src/vmm)
    Checking virtio_gen v0.1.0 (/firecracker/src/virtio_gen)
   Compiling firecracker v1.5.0-dev (/firecracker/src/firecracker)
   Compiling jailer v1.5.0-dev (/firecracker/src/jailer)
   Compiling rebase-snap v1.5.0-dev (/firecracker/src/rebase-snap)
   Compiling cpu-template-helper v1.5.0-dev (/firecracker/src/cpu-template-helper)
    Checking snapshot v0.1.0 (/firecracker/src/snapshot)
error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
   --> src/net_gen/src/iff.rs:787:1
    |
787 | / pub struct if_settings {
788 | |     pub type_: ::std::os::raw::c_uint,
789 | |     pub size: ::std::os::raw::c_uint,
790 | |     pub ifs_ifsu: if_settings__bindgen_ty_1,
791 | | }
    | |_^
    |
    = note: requested on the command line with `-D missing-debug-implementations`

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
   --> src/net_gen/src/iff.rs:794:1
    |
794 | / pub union if_settings__bindgen_ty_1 {
795 | |     pub raw_hdlc: *mut raw_hdlc_proto,
796 | |     pub cisco: *mut cisco_proto,
797 | |     pub fr: *mut fr_proto,
...   |
801 | |     pub te1: *mut te1_settings,
802 | | }
    | |_^

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
    --> src/net_gen/src/iff.rs:1019:1
     |
1019 | / pub struct ifreq {
1020 | |     pub ifr_ifrn: ifreq__bindgen_ty_1,
1021 | |     pub ifr_ifru: ifreq__bindgen_ty_2,
1022 | | }
     | |_^

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
    --> src/net_gen/src/iff.rs:1025:1
     |
1025 | / pub union ifreq__bindgen_ty_1 {
1026 | |     pub ifrn_name: [::std::os::raw::c_uchar; 16usize],
1027 | | }
     | |_^

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
    --> src/net_gen/src/iff.rs:1069:1
     |
1069 | / pub union ifreq__bindgen_ty_2 {
1070 | |     pub ifru_addr: sockaddr,
1071 | |     pub ifru_dstaddr: sockaddr,
1072 | |     pub ifru_broadaddr: sockaddr,
...    |
1082 | |     pub ifru_settings: if_settings,
1083 | | }
     | |_^

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
    --> src/net_gen/src/iff.rs:1385:1
     |
1385 | / pub struct ifconf {
1386 | |     pub ifc_len: ::std::os::raw::c_int,
1387 | |     pub ifc_ifcu: ifconf__bindgen_ty_1,
1388 | | }
     | |_^

error: type does not implement `std::fmt::Debug`; consider adding `#[derive(Debug)]` or a manual implementation
    --> src/net_gen/src/iff.rs:1391:1
     |
1391 | / pub union ifconf__bindgen_ty_1 {
1392 | |     pub ifcu_buf: *mut ::std::os::raw::c_char,
1393 | |     pub ifcu_req: *mut ifreq,
1394 | | }
     | |_^

error: could not compile `net_gen` (lib) due to 7 previous errors
warning: build failed, waiting for other jobs to finish...
error: could not compile `net_gen` (lib test) due to 7 previous errors
root@a71d948ddcbf:/firecracker# 
```

Reply:
Oh, ok.